applications/machine_learning/src/modules/Kconfig.usb_state

```diff
@@ -12,6 +12,14 @@ menuconfig ML_APP_USB_STATE
 	help
 	  Enable USB subsystem on application start.
 
+if ML_APP_USB_STATE
```

Comment:
I think we should consider disabling the `CONFIG_BOARD_SERIAL_BACKEND_CDC_ACM` in Thingy:53 MCUboot configuration. Actually it should not be needed for the MCUboot

Reply:
We also should definitely disable one of the USB CDC ACM instances in Thingy:53 MCUboot's DTS (after your changes you have 2 USB CDC instanced and one of them is unused)

Reply:
On instance is added by Thingy:53 board and the second one comes from `usb.overlay` applied automatically in the `nrf/modules/mcuboot/CMakelists.txt` file

Reply:
Discussed offline, taken steps:
+ sdk-mcuboot noup change where `CONFIG_BOARD_SERIAL_BACKEND_CDC_ACM` is disabled in thingy board config https://github.com/nrfconnect/sdk-mcuboot/pull/237
+ sdk-zephyr noup change where `CONFIG_MCUBOOT_USB_SUPPORT` is disabled in thingy board config https://github.com/nrfconnect/sdk-zephyr/pull/1043